tests/conftest.py

```diff
@@ -453,3 +455,24 @@ def invalid_classname_xml(tmpdir_factory):
         f.write(junit_xml)
 
     return filename
+
+
+@pytest.fixture(scope='session')
+def single_test_with_jira_tickets_xml(tmpdir_factory):
```

Comment:
This fixture doesn't appear to be necessary because the update to `DEFAULT_TESTCASE_PROPERTIES` affects all fixtures that use the default. You can just use the `single_passing_xml` fixture instead.

Reply:
Removed